client_cli/pytests/README.md

```diff
@@ -0,0 +1,88 @@
+# Overview
+
+This directory contains the pytest framework with test suites for Iroha 2's Client CLI.
+
+## Iroha 2 Test Model
+The Iroha 2 Test Model consists of several test categories that cover different aspects of the Iroha 2 blockchain platform. The structure of the test model is as follows:
+
+- **Configurations:** Test configurations for the Iroha 2 platform.
+
+- **Accounts:** Test cases for account-related operations, such as account registration, key management, and metadata manipulation.
+
+- **Assets:** Test cases for asset-related operations, including asset creation, minting, burning, transferring, and managing asset definitions and metadata.
+
+- **Domains:** Test cases for domain-related operations, such as registering and unregistering domains.
+
+- **Roles:** Test cases for roles management.
+
+- **Atomicity:** Test cases for transaction atomicity, including multiple instructions within a single transaction, paired instructions, and invalid instructions.
+
+## How to use
+At first, you need to installed and running [Iroha 2](https://hyperledger.github.io/iroha-2-docs/guide/install.html), and also need to have built [Client CLI](https://hyperledger.github.io/iroha-2-docs/guide/build.html)
+
+## Configuration
+
+To configure the application, you can use a `.env` file in the `client_cli/pytest` directory. The `.env` file should contain the following variables:
+
+```
+CLIENT_CLI_DIR=/path/to/iroha_client_cli/dir/
```

Comment:
what are the contents of this directory? Is it both `iroha_client_cli` and `config.json`?